ffcx/ir/elementtables.py

```diff
@@ -54,7 +54,7 @@ def clamp_table_small_numbers(table,
 
 
 def get_ffcx_table_values(points, cell, integral_type, ufl_element, avg, entitytype,
-                          derivative_counts, flat_component):
+                          derivative_counts, flat_component, finite_element_args):
```

Comment:
Add some docs - not sure what `finite_element_args` is.

Reply:
Thankfully, `finite_element_args` is no longer necessary so the code is now a little simpler